README.md

```diff
@@ -119,6 +119,20 @@ We welcome community contributions to this project.
 
 Please read our [Contributor Guide](CONTRIBUTING.md) for more information on how to get started.
 
+## Related Projects
```

Comment:
We should describe a bit more what this is also. something like this could work

```suggestion
## Related Projects

Historical and other related projects for compiling Rust code to GPUs.

```

Reply:
Updated with suggestion